src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -117,7 +118,7 @@
     :title       (i18n/label :t/invite-friends-to-status)
     :description (i18n/label :t/share-invite-link)}})
 
-(defn home
+(defn f-view-internal
```

Comment:
so here you will now have access to `theme` prop and we should pass this to `theme-value` function as it's 3rd param.

i.e 
```
(defn f-view-internal
  [{:keys [theme] ]
  ...
```

and then 
`(theme/theme value-1 value-2 theme)`